src/components/Hero/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { IGatsbyImageData } from "gatsby-plugin-image"
+import { CTAParentProps } from "./CallToAction"
+
+export type CommonHeroProps = {
+  heroImgSrc: IGatsbyImageData
+  header: string
+  title: string
+  description: string
+  buttons?: [CTAParentProps, CTAParentProps?]
```

Comment:
Want to point this out to make sure intention is clear.
This typing for `buttons`, where...
- The prop itself is `optional`
- It specifies a tuple of the type `CTAParentProps`
- The second "element" of the tuple is `optional`

means that 0 - 2 buttons can be rendered but no more than two, as the design calls for only up to two. Else a type error will throw.